swagger.yaml

```diff
@@ -6028,6 +6300,79 @@ components:
         - asset: "b0d07d45fe9514f80213f4020e5a61241458be626841cde717cb574636f696e"
           quantity: "18605647"
 
+    scripts:
+      type: array
+      items:
+        type: object
+        properties:
+          script_hash:
+            type: string
+            description: Script hash
+        required:
+          - script_hash
+      example:
+        - script_hash: "13a3efd825703a352a8f71f4e2758d08c28c564e8dfcce9f77776ad1"
+        - script_hash: "e1457a0c47dfb7a2f6b8fbb059bdceab163c05d34f195b87b9f2b30e"
+        - script_hash: "a6e63c0ff05c96943d1cc30bf53112ffff0f34b45986021ca058ec54"
+
+    script:
+      type: object
+      properties:
+        script_hash:
+          type: string
+          example: "13a3efd825703a352a8f71f4e2758d08c28c564e8dfcce9f77776ad1"
+          description: Script hash
+        type:
+          type: string
+          enum: [timelock, plutus]
+          example: "plutus"
+        serialised_size:
+          type: integer
+          nullable: true
+          example: 3119
```

Comment:
Description missing here as well, maybe also `seriali(z)ed_size`?

Reply:
Serialised is mostly used in the specs, leaving as it is, tribute to Duncan. :)

Reply:
UK spelling.